standalone-metastore/metastore-server/src/main/sql/derby/hive-schema-4.0.0-alpha-2.derby.sql

```diff
@@ -558,7 +558,8 @@ CREATE TABLE TXN_COMPONENTS (
   TC_TABLE varchar(256),
   TC_PARTITION varchar(767),
   TC_OPERATION_TYPE char(1) NOT NULL,
-  TC_WRITEID bigint
+  TC_WRITEID bigint,
+  TC_ID bigint PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY
```

Comment:
TC_ID can't be a PRIMARY KEY as it's not unique. TXN_COMPONENTS might have a bunch of partitions updated/inserted in the same txn.